### DISCUSSION TITLE
Execution of spring-messagesource-xliff

### DISCUSSION THREAD

**Discussion by AstghikG7** [alaugks/spring-messagesource-xliff#6] [General]
Error in finding the main

**Comment by alaugks** [comment]
From our communication before, I can see that you have checked out [spring-messagesource-xliff](https://github.com/alaugks/spring-messagesource-xliff]https://github.com/alaugks/spring-messagesource-xliff) and [spring-messagesource-xliff-example](https://github.com/alaugks/spring-messagesource-xliff-example).

**spring-messagesource-xliff** is just the package you need to define as a dependency (Maven or Gradle) in your Spring Boot application, which is then loaded by Maven Central. You can delete this repository from your local computer. 

**spring-messagesource-xliff-example** is a minimal Spring Boot example application that has the package included.

I have no experience with Netbeans. But please try the following on the command line. Important: You need to have Maven installed.

1. On the command line change to the root directory of the project.

2. Install all depenedcies:

```bash
mvn clean install
```

3. Rund Spring boot Application

```bash
mvn spring-boot:run
```

4. Application run on Port 8080. Open in Browser http://localhost:8080

**Comment by AstghikG7** [comment]
Now it works! Thank you so much for your help
________________________________
De: André Laugks ***@***.***>
Enviat el: dimecres, 7 d’agost de 2024 17:57
Per a: alaugks/spring-messagesource-xliff ***@***.***>
A/c: Astghik Galstyan Minasyan ***@***.***>; Author ***@***.***>
Tema: Re: [alaugks/spring-messagesource-xliff] Execution of spring-messagesource-xliff (Discussion #6)


From our communication before, I can see that you have checked out spring-messagesource-xliff<https://github.com/alaugks/spring-messagesource-xliff%5Dhttps://github.com/alaugks/spring-messagesource-xliff> and spring-messagesource-xliff-example<https://github.com/alaugks/spring-messagesource-xliff-example>.

spring-messagesource-xliff is just the package you need to define as a dependency (Maven or Gradle) in your Spring Boot application, which is then loaded by Maven Central. You can delete this repository from your local computer.

spring-messagesource-xliff-example is a minimal Spring Boot example application that has the package included. Can you run this app location?

I have no experience with Netbeans. But please try the following on the command line. Important: You need to have Maven installed.

  1.  On the command line change to the root directory of the project.

  2.  Install all depenedcies:

mvn clean install

  1.  Rund Spring boot Application

mvn spring-boot:run

  1.  Application run on Port 8080. Open in Browser http://localhost:8080
